.editorconfig

```diff
@@ -24,81 +24,81 @@ dotnet_style_require_accessibility_modifiers = always:error
 csharp_style_var_when_type_is_apparent = true:error
 csharp_prefer_braces = true:error
 csharp_using_directive_placement = outside_namespace:error
-
```

Comment:
Don't think we should comment this out?

Reply:
Didn't actually mean to check this in, but... there's so much code in mlapi right now that isn't following these standards that it's causing the signal to noise ratio to be extremely high when trying to identify actual errors. It's kind of annoying to have to manually build or go back to the unity window to know if my code compiles or not. 

Personally I think if we're going to make naming trigger errors, we should fix them... if we're not going to fix them, the errors are providing more annoyance than value.

Reply:
This was reverted.